diff --git a/arm64intr.h b/arm64intr.h
index 4943b2d..4b42e76 100644
--- a/arm64intr.h
+++ b/arm64intr.h
@@ -15,7 +15,7 @@
 #ifndef __ARM64INTR_H
 #define __ARM64INTR_H
 
-typedef enum
+/*typedef enum
 {
   _ARM64_BARRIER_SY    = 0xF,
   _ARM64_BARRIER_ST    = 0xE,
@@ -29,7 +29,7 @@ typedef enum
   _ARM64_BARRIER_OSH   = 0x3,
   _ARM64_BARRIER_OSHST = 0x2,
   _ARM64_BARRIER_OSHLD = 0x1
-} _ARM64INTR_BARRIER_TYPE;
+} _ARM64INTR_BARRIER_TYPE;*/
 
 #endif /* __ARM64INTR_H */
 #endif /* _MSC_VER */
